### PR TITLE
DOC: slight edits

### PR DESCRIPTION
**Pull request by jklymak**
Here are few more.   

Does the metrics/ outcomes need more?

### CODE REVIEW

File: CZI_mpl.tex [hunk 1]
```diff
@@ -55,7 +55,7 @@ \section{Goals}
 
 
 % The goal is the maintenance; curating is the method of achieving it.
-\subsection{Maintenance of the library}
+\subsection{Revitalizing the developer community}
```
Comment (efiring) [inline CZI_mpl.tex:58]: "Revitalizing" implies a lack of vitality, which I think is misleading, or at least not the tone we want to convey.  Maybe "Enlarging"? Or "Cultivating"?  The point of the latter is that we need more efficient and effective development, and this is far from being just a matter of size (number of contributors).

Comment (jklymak) [reply CZI_mpl.tex:58]: I like cultivating. My point here is that “Maintenance” might not sound like something they want to fund.  OTOH if other proposals have been successful proposing maintenance as a big part of the proposal, then maybe this concern is misplaced

Comment (tacaswell) [reply CZI_mpl.tex:58]: CZI specifically calls out maintenance in the program description:  "The goal of the program is to support software maintenance, growth, development, and community engagement for these critical tools."  and this funding path was conceived in part to address the gap left by other funding sources.

Comment (jklymak) [reply CZI_mpl.tex:58]: OK, "Maintenance and Growth of the Library"?

Comment (tacaswell) [reply CZI_mpl.tex:58]: "Maintenance and Growth of Library and Community" ?

**Pull request merged**